### PR TITLE
Import Aws service package and pass EndpointsID to serviceController

### PR DESCRIPTION
**Pull request by ryansteakley**
Issue #, if available:
https://github.com/aws-controllers-k8s/community/issues/896

Description of changes:
Import service and get its EndpointsID to pass this into the NewServiceController

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: templates/cmd/controller/main.go.tpl
Comment (RedbackThomson) [inline templates/cmd/controller/main.go.tpl:17]: We usually import as `svcsdk` then we can just do `svcsdk.EndpointsID` later - see https://github.com/aws-controllers-k8s/code-generator/blob/main/templates/pkg/resource/sdk.go.tpl#L14

Comment (ryansteakley) [reply templates/cmd/controller/main.go.tpl:17]: cool, will change to be consistent

**Review by A-Hilaly — APPROVED** [review]

**Comment by jaypipes** [comment]
/hold

@ryansteakley please update `go.mod` to refer to ACK runtime 0.11.0, which I just cut. Don't forget to update Makefile's VERSION, too.

I've put a hold on this until we merge https://github.com/aws-controllers-k8s/code-generator/pull/138 which will also modify go.mod to update to ACK runtime v0.10.0...

**Pull request merged**
